drivers/hmis/app/models/hmis/form/definition_validator.rb

```diff
@@ -10,23 +10,47 @@ def self.perform(...)
   end
 
   # @param [Hash] document is a form definition document {'item' => [{...}] }
-  # @param [Hash] valid_pick_lists validate that pick list references are in this set
-  def perform(document, valid_pick_lists: [])
-    @issues = []
-    all_ids = check_ids(document, valid_pick_lists.to_set)
+  # @param [role] role of the form as string ('INTAKE', 'CLIENT', etc). If not provided, HUD rule validation will not occur.
+  def perform(document, _role = nil)
```

Comment:
suggest taking out role since it's kind of misleading (unless we are implementing it very soon)

Reply:
Oh I see, we have this in 4475.  Ignore my comment above